addons/purchase_stock/models/stock.py

```diff
@@ -113,11 +113,13 @@ def _get_global_route_rules_values(self):
                     'picking_type_id': self.in_type_id.id,
                     'group_propagation_option': 'none',
                     'company_id': self.company_id.id,
+                    'propagate': self.reception_steps != 'one_step',
```

Comment:
you don't need to add the both key in create and update since the method that use the dict will do an
create_values.update(update_values). Keep only the key in update